kubeflow/hack/create_context.sh

```diff
@@ -6,24 +6,12 @@
 # PROJECT=<PROJECT> REGION=<region> NAME=<NAME>
 #
 # TODO(jlewi): Support zonal clusters as well
-set -x 
+set -ex
 
 # Default namespace to kubeflow
 NAMESPACE=${NAMESPACE:-kubeflow}
-echo Checking if context ${NAME} exists 
```

Comment:
I believe you need to delete the context if it already exists

Reply:
Oh yes!
I was hoping to find an option like `--override` or `--force` in the rename context command. But unfortunately could not find. So using the previous conditional procedure again for checking the context.